### ISSUE TITLE
PNG images are not being displayed.

### ISSUE THREAD

**Issue by Krishna16** [smarteist/Android-Image-Slider#107]
Hey guys. So I was trying out this slider view and noticed that my PNG images which I have stored in the drawable folder aren't getting displayed. When I write ("" + R.drawable.save) in the setImageUrl method, it just displays a white screen. I tried adding the url method in the addNewItems method and renewItems as well, with no luck. Has anyone else faced this issue? Thanks in advance.

**Comment by erwandy92** [comment]
Try deleting the cache on your App and restart. See if it works. I faced the same white screen problem also before.

**Comment by Krishna16** [comment]
> Try deleting the cache on your App and restart. See if it works. I faced the same white screen problem also before.

Hey, thanks for your response, but unfortunately it didn't solve my problem. Tell me one thing, did you need to adjust the size of your images?

**Comment by erwandy92** [comment]
I don't think so. For the sizing matter, I only adjusted it from XML through constraintLayout. 
For my case, I display the gif and jpg images. But, basically png ones should be no problem, too. 

Do u use library to load the image? If not, you can try using Glide. It's quite powerful in handling various types of image.

**Comment by smarteist** [comment]
It should be related with ur layout or image loader .

**Comment by Krishna16** [comment]
> It should be related with ur layout or image loader .

I didn't get you. Can you be more specific?

**Comment by yadavpriya73028** [comment]
How to add drawable pics in auto slider ...I can try (""+R.drawable.pic) but still it not work:'( only white screen displayed
